docs/source/openmw-cs/tour.rst

```diff
@@ -13,7 +13,7 @@ installed and playable.
 Adding the ring to the game's records
 *************************************
 
-In this first section we will define what our new ring is, what it looks like
+In this first section we will define what our new ring is, what it looks like,
```

Comment:
That extra comma needs to go, it should not be there. The APA and MLA can go get stuffed, the rest of the English speaking world is against the serial comma.